Week-2/form-assignment/index.html

```diff
@@ -0,0 +1,87 @@
+
+<html>
+<head>
+    <link href="base.css" type="text/css" rel="stylesheet">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
```

Comment:
@Dookiee The content is not responsive. Can you please check ?

<img width="451" alt="Screenshot 2022-08-06 at 12 39 05 PM" src="https://user-images.githubusercontent.com/9133593/183238615-51049edb-132e-4527-be12-27578b4293ac.png">

Reply:
issue fixed!
<img width="776" alt="Screenshot 2022-08-06 at 11 55 44 PM" src="https://user-images.githubusercontent.com/8068468/183261403-dc7132be-fb9f-4e6a-9832-355bd28beb90.png">